PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h

```diff
@@ -2499,7 +2239,8 @@ void CalculateEverything()
 
 //============================================================
 
-void SteerRec(CollisionRec const& collision, TracksRec const& tracks)
+template <typename T1, typename T2>
+void Steer(T1 const* collision, T2 const* tracks)
```

Comment:
Consider templating also over the reco sim `enum`  that will simplify a lot your code and you just pass it to your function calls

Reply:
See my reply above - I am not in general against templates, but I do not want literally to have only templates in my code.